src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/FinalClass/newline/defaultContext.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/Test.java b/Test.java
+index 2fc8329..d3dd3f2 100644
+--- a/Test.java
++++ b/Test.java
```

Comment:
`newline/expected.txt` should have at least one violation, always, please update patch .
Please update all other PRs.